pkgs/applications/audio/spotify/update.sh

```diff
@@ -38,13 +38,14 @@ snap_info=($(
 # "revision" is the actual version identifier on snapcraft, the "version" is
 # just for human consumption. Revision is just an integer that gets increased
 # by one every (stable or unstable) release.
+
 revision="${snap_info[0]}"
-sha512="${snap_info[1]}"
+# We need to escape the slashes
+hash="$(nix hash to-sri --type sha512 ${snap_info[1]} | sed 's/\//\\\//g')"
 upstream_version="${snap_info[2]}"
 last_updated="${snap_info[3]}"
 
 echo "Latest $channel release is $upstream_version from $last_updated."
-
```

Comment:
I would undelete to minimise diff

Reply:
What do you want me to do here, I don't get it